packages/rescript-relay-router/.gitignore

```diff
@@ -20,3 +20,6 @@ stats.html
 # Commit our Vite plugins written in plain JavaScript.
 !vite-plugins/**/*.mjs
 !test/RescriptRelayServerVitePlugin.test.mjs
+
+# commit our bundled files
+!*.bundle.mjs
```

Comment:
@tsnobip hmm, why are the bundled files commited?

Reply:
Hmm, it was just to publish them, but we can publish them without versioning them I guess indeed!

Reply:
That'd be great I think, if we can just build them before publishing and not have them commited.